app/views/documents/_index.html.erb

```diff
@@ -21,8 +21,8 @@
               <table id="documents" class="table table-sm table-hover">
                 <thead>
                   <tr>
-                    <th scope="col">Nom du document</th>
-                    <th scope="col">Télécharger</th>
+                    <th scope="col"> <%= t 'activerecord.attributes.document.filename' %></th>
+                    <th scope="col"><%= t 'documents.download_file' %></th>
```

Comment:
You can use the I18n lookup shortcut here, by prefixing the string with a `.`
It's actually pretty useful to get some guidance on how to split our locale files and how to organize our locale keys.
For example, here if you do `t '.download_file'`, you'll have a 'missing translation' on the `documents.index.download_file` (guessing here). Doing so would make it clear that the locale keys we need here are only scoped to this view